src/map/status.c

```diff
@@ -12319,6 +12326,7 @@ void status_defaults(void) {
 	status->calc_mob_ = status_calc_mob_;
 	status->calc_pet_ = status_calc_pet_;
 	status->calc_pc_ = status_calc_pc_;
+	status->calc_pc_additional = status_calc_pc_additional;
```

Comment:
Even we add this, codes put into it will not work properly...bonuses inside 'status_calc_pc_' is arrange sequentially...so adding custom bonuses some won't work....